weblate/accounts/models.py

```diff
@@ -496,6 +496,30 @@ class Profile(models.Model):
         default=False
     )
 
+    DASHBOARD_SUBSCRIPTIONS = "your-subscriptions"
+    DASHBOARD_LANGUAGES = "your-languages"
+    DASHBOARD_ALL = "projects"
+    DASHBOARD_COMPONENT_LIST = "list"
+    DASHBOARD_CHOICES = (
+        (DASHBOARD_SUBSCRIPTIONS, _('Your subscriptions')),
+        (DASHBOARD_LANGUAGES, _('Your languages')),
+        (DASHBOARD_ALL, _('All projects')),
+        (DASHBOARD_COMPONENT_LIST, _('Component list')),
+    )
+    dashboard_view = models.CharField(
+        max_length=100,
+        choices=DASHBOARD_CHOICES,
+        verbose_name=_('Default dashboard view'),
+        default=DASHBOARD_SUBSCRIPTIONS,
+    )
```

Comment:
The problem is that these strings are listed explicitly in index.html, and also show up in URLs. "dashboard-1" would be pretty ugly for one, and confusing in the html code too. So we actually need the symbolic names _and_ the slug values. That is why we chose to put the slug values in the database directly; there is actually no place where numeric id is of any use.

We could decide to ignore the "ugly URL" thing, and use symbolic names in the html source, but that doesn't seem to lead to clarity either: `<a href="#dashboard-{{ Profile.DASHBOARD_SUBSCRIPTIONS}}">` is worse than `<a href="#subscriptions">` IMHO.

A possibility is to use a translation table between ids and slugs, or some sort of function to return slug given the ID, or label given the ID, and a list of slugs from which to generate the drop-down menu... I don't have a clear idea how to do this and where to put the resulting code though.
Hints? :)

Note that we expect the `active_tab` properties to be reusable on other pages, namely settings (#990). That's why we chose the generic names.